pyvista/plotting/plotting.py

```diff
@@ -1387,6 +1387,16 @@ def add_mesh(self, mesh, color=None, style=None, scalars=None,
             If set to ``True``, then the number of unique values in the scalar
             array will be used as the ``n_colors`` argument.
 
+        silhouette : dict, bool, optional
+           As a dict, it contains the properties of the silhouette to display:
+               * ``color`` (color of the silhouette)
+               * ``line_width`` (edge width)
+               * ``opacity`` (edge transparency between 0 and 1)
+               * ``feature_angle`` (bool to display sharp edges)
```

Comment:
Some of these features, like `feature_angle`, are not entirely obvious. Perhaps it would be useful to add an example that shows what it does?

Reply:
You mean like a full fledge example? In this case, I could re-use the on in the PR

Reply:
Yeah, I think it would fit nicely in https://docs.pyvista.org/examples/#plotting. Actually most of the options seem straightforward, but if the PR's example shows what `feature_angle` does it would be pretty neat.